containers/kubernetes/reference-content/version-support-policy.mdx

```diff
@@ -89,4 +102,34 @@ The following documentation lists only the main feature changes for each version
 * Exec Probe Timeout Handling
 
 ### 1.19
-Official [release announcement](https://kubernetes.io/blog/2020/08/26/kubernetes-release-1.19-accentuate-the-paw-sitive/)
\ No newline at end of file
+Official [release announcement](https://kubernetes.io/blog/2020/08/26/kubernetes-release-1.19-accentuate-the-paw-sitive/)
+
+## Supported Container Runtime Interfaces (CRI)
+
+### Kubernetes v1.26
+Starting from Kubernetes version 1.26 launch:
+* Kubernetes Kapsule only supports **cilium** and **calico** as CNI.
```

Comment:
This whole part refers to the CNI, not the CRI.
Lines 111 to 122 need to move under 
## Supported Container Network Interfaces (CNI)